service/job/job_controller.py

```diff
@@ -71,6 +76,20 @@ def job_status(job_id):
     return jsonify(response)
 
 
+@job_controller.route('/jobtypes', methods=['GET'])
```

Comment:
Ich würde die Route `/type` nennen. Dann hat man nicht die Doppelung von "job" und man kann in guter REST-Tradition den Endpoint für die Infos zum einzelnen Job-Type unter `/type/<job_type>` ansiedeln.

Reply:
"type" und nicht "types" ?

Reply:
Ja, wobei das auch nur persönliche Präferenz ist. Siehe: https://stackoverflow.com/questions/6845772/rest-uri-convention-singular-or-plural-name-of-resource-while-creating-it

Wie so oft gilt: Ich habe da keine starke Meinung, es muss nur konsistent sein.

Reply:
Alternativer Vorschlag:

    POST /jobs/<job_type> Job anlegen
    GET /jobs Alle jobs des Users anzeigen (gibts noch nicht)
    GET /jobs/<job_id> Job status abfragen
    DELETE /jobs/<job_id> Job löschen / abbrechen (gibts noch nicht)
    GET /job_types Alle Job-Types anzeigen
    GET /job_types/<job_type> Infos zu einem Job-Type anzeigen

Das würde bedeuten, dass wir am besten auch einen separaten `JobTypeController` anlegen.

Reply:
`GET /job/jobtypes` kann doch aktuell auch nicht richtig funtkionieren, oder? Ich würde jedenfalls vermuten, dass sich das mit `GET /job/<job_id>` in die Quere kommt.